client-guide/heartbeat-meeting.html.md

```diff
@@ -5,7 +5,7 @@ meta_description: >
   Learn how we keep track of projects' health!
 ---
 
-We aim to have this meeting quarterly. You will talk with our Team lead about our relationship.
+We aim to have this meeting at least quarterly. You will talk with our Team lead about our relationship.
```

Comment:
This isn't a practice anymore either actually... We talk to clients all the time, we realized there's no need for fixed checkpoints. Can you remove this as well?